### PR TITLE
Add mirroring to quay for network-metrics-daemon

### PR DESCRIPTION
**Pull request by fedepaol**

### CODE REVIEW

**Comment by fedepaol** [comment]
@smarterclayton , would you mind giving a look at this?
I think you are the only one with the right approval permissions.

File: core-services/image-mirroring/openshift/mapping_origin_4_5 [hunk 1]
```diff
@@ -163,6 +163,7 @@ registry.svc.ci.openshift.org/origin/4.5:multus-cni quay.io/openshift/origin-mul
 registry.svc.ci.openshift.org/origin/4.5:multus-route-override-cni quay.io/openshift/origin-multus-route-override-cni:4.5 quay.io/openshift/origin-multus-route-override-cni:4.5.0 quay.io/openshift/origin-multus-route-override-cni:latest
 registry.svc.ci.openshift.org/origin/4.5:multus-whereabouts-ipam-cni quay.io/openshift/origin-multus-whereabouts-ipam-cni:4.5 quay.io/openshift/origin-multus-whereabouts-ipam-cni:4.5.0 quay.io/openshift/origin-multus-whereabouts-ipam-cni:latest
 registry.svc.ci.openshift.org/origin/4.5:must-gather quay.io/openshift/origin-must-gather:4.5 quay.io/openshift/origin-must-gather:4.5.0 quay.io/openshift/origin-must-gather:latest
+registry.svc.ci.openshift.org/origin/4.5:network-metrics-daemon quay.io/openshift/origin-network-metrics-daemon:4.5 quay.io/openshift/origin-network-metrics-daemon:4.5.0 quay.io/openshift/origin-network-metrics-daemon:latest
```
Comment (smarterclayton) [inline core-services/image-mirroring/openshift/mapping_origin_4_5:166]: Will *every* cluster have these metrics?  I.e. no matter what these metrics will always be returned in every 4.5 cluster?  Or are the metrics within this deployed only in specific scenarios?

Comment (fedepaol) [reply core-services/image-mirroring/openshift/mapping_origin_4_5:166]: @smarterclayton  as things are today, every cluster would have these metrics. 
The current idea is to have CNO ship this under the same parameter that deploys multus (which is enabled by default).

Comment (fedepaol) [reply core-services/image-mirroring/openshift/mapping_origin_4_5:166]: @smarterclayton does this make sense? Can we push this forward?

**Pull request closed without merging**